e2e/testcases/v2-advanced-search/8-search-death-location.spec.ts

```diff
@@ -1,26 +1,23 @@
 import { expect, test, type Page } from '@playwright/test'
 import { loginToV2 } from '../../helpers'
 
-test.describe
-  .serial("Advanced Search 8 - Death - Deceased's place of death", () => {
-  let page: Page
+const assertPills = async (page: Page, pills: string[]) => {
```

Comment:
I think a similar assertion is done on other advanced search tests, this could be shared with those other tests